srcpkgs/python-markdown2/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'python-markdown2'
+pkgname=python-markdown2
+version=2.3.6
+revision=1
+noarch=yes
+build_style=python-module
+pycompile_module="markdown2"
+hostmakedepends="python-setuptools python3-setuptools"
+short_desc="A fast and complete implementation of Markdown in Python2"
+maintainer="Aluísio Augusto Silva Gonçalves <aluisio@aasg.name>"
+license="MIT"
+homepage="https://github.com/trentm/python-markdown2"
+distfiles="https://github.com/trentm/python-markdown2/archive/${version}.tar.gz"
+checksum=@ce50839cc8b30c448adfd2873782bd25753ee5a93ac605f7335ac4f0132d78d2
+
+post_install() {
+	vlicense LICENSE.txt
+}
+
+python3-markdown2_package() {
+	noarch=yes
+	pycompile_module="markdown2"
+	short_desc="${short_desc/Python2/Python3}"
+	pkg_install() {
+		vmove usr/lib/python3*
```

Comment:
the path should be quoted.

Reply:
Done here and in another package as well.